technical/validations.md

```diff
@@ -7,20 +7,35 @@ these validations will be duplicated in the front-end code for an optimal user e
 ## Entity-level validations
 
 ### Playground
-- The name of the playground is between 3 en 40 characters
-- Two playgrounds cannot be located within .. metres of each other
+- The name of the playground is between 3 en 40 characters ( no whitespaces in the beginning and trailing of the playground name )
+- Two playgrounds cannot be located within .. metres of each other ( Are there any API/Service provided by Netherlands to verify entered/chosen address )
 - The maximum number of playgrounds that can exist within the system is ..
+- At least 4(TBD) decimal precisions required for Longitude and Latitude of the Playground location, in order to identify how close two Playgrounds are located
+- A User must not be able to join a Playground twice
+- The id, name, lat, lng and status are required properties during creation of a new Playground
+- The smokeFreeDate must be Future or Present Date
 
 ### Playground workspace
 - The maximum number of volunteers that can enter a playground workspace is 200
 - The maximum number of managers in a workspace is ..
 
-
 ### Chat message
 - a chat message cannot consist of whitespace only
 - the minimum length of a chat message is 1 characters
 - the maximum length of a chat message is 300 characters
 - this is the set of valid characters for a chat message: ...  (or provide a regexp)
 
+### Playground Observation
+- A comment message can't consist of whitespaces only, and no whitespaces in the beginning or/and trailing of the comment
+- The minimum length of a comment message is 100 characters
+- The maximum length of a comment message is ... ( The maximum characters allowed in MySQL TEXT Data Type is 65k )
+- How frequently a Volunteer can indicate his observations about The Playground
```

Comment:
Please use consistent capitalisation. I think we should not capitalise the terms